core/src/main/java/hudson/model/queue/Executables.java

```diff
@@ -61,14 +62,19 @@
 
     /**
      * Returns the estimated duration for the executable.
+     * If the Executable is null the Estimated Duration can't be evaluated, then -1 is returned.
```

Comment:
for future reference: when using code literals like `null` `true` false`  `magic_number` wrap them in `{@code wibble}` so they get rendered differently in the javadoc